src/controllers/reactWebviewBaseController.ts

```diff
@@ -239,6 +239,17 @@ export abstract class ReactWebviewBaseController<State, Reducers>
                 return undefined;
             }
         };
+
+        this._webviewRequestHandlers["executeCommand"] = async (message) => {
+            if (!message?.command) {
+                console.log("No command provided to execute");
```

Comment:
if there's no command, should this `return` instead of attempting to execute a commandless command?

Reply:
forgot about this when I changed from `throw` to `console.log` :), updated